Src/CSharpier/IgnoreFile.cs

```diff
@@ -0,0 +1,78 @@
+using System;
+using System.IO.Abstractions;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace CSharpier
+{
+    public class IgnoreFile
+    {
+        protected global::Ignore.Ignore Ignore { get; }
```

Comment:
Where is `global::Ignore.Ignore` defined in? Is it part of some weird library? Some ancient framework API?